pom.xml

```diff
@@ -62,7 +62,7 @@ limitations under the License.
         <protobuff-java.version>3.5.1</protobuff-java.version>
         <google.api.client.version>1.23.0</google.api.client.version>
         <google.http.client.version>1.23.0</google.http.client.version>
-        <guava.version>19.0</guava.version>
+        <guava.version>20.0</guava.version>
```

Comment:
Please keep guava at 19.0

Reply:
If I keep Guava 19.0 then I get below error for integration test execution -

Results :

Failed tests: 
  TestAsyncAdmin.testCreateTable_exception 
Expected: an instance of java.lang.NullPointerException
     but: <java.lang.UnsupportedOperationException: createTable> is a java.lang.UnsupportedOperationException
Stacktrace was: java.lang.UnsupportedOperationException: createTable
	at com.google.cloud.bigtable.hbase2_x.BigtableAsyncAdmin.createTable(BigtableAsyncAdmin.java:685)
	at com.google.cloud.bigtable.hbase.async.TestAsyncAdmin.testCreateTable_exception(TestAsyncAdmin.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

Tests in error: 
  TestAsyncAdmin.testCreateTable_harness:86 » UnsupportedOperation createTable
  TestAsyncAdmin.testDisbleTable_exceptions:213 » UnsupportedOperation createTab...
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndDeleteDiffQual:177->checkAndDelete:81->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndDeleteDiffRow:242->checkAndDelete:81->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndDeleteSameQual:105->checkAndDelete:81->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndMutate:264->checkAndMutate:91->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndPutDiffQual:136->checkAndPut:61->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndPutDiffRow:219->checkAndPut:61->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCheckAndPutSameQual:54->checkAndPut:61->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCompareOps:313->checkAndPut:71->getBuilder:99 » NoClassDefFound
  TestAsyncCheckAndMutate>AbstractTestCheckAndMutate.testCompareOpsVersions:386->checkAndPut:71->getBuilder:99 » NoClassDefFound

Tests run: 163, Failures: 1, Errors: 11, Skipped: 1

Reason is below 

-------------------------------------------------------------------------------
Test set: com.google.cloud.bigtable.hbase.async.TestAsyncCheckAndMutate
-------------------------------------------------------------------------------
Tests run: 9, Failures: 0, Errors: 9, Skipped: 0, Time elapsed: 0.317 sec <<< FAILURE! - in com.google.cloud.bigtable.hbase.async.TestAsyncCheckAndMutate
testCheckAndDeleteDiffRow(com.google.cloud.bigtable.hbase.async.TestAsyncCheckAndMutate)  Time elapsed: 0.008 sec  <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/hadoop/hbase/shaded/com/google/common/base/Preconditions
	at com.google.cloud.bigtable.hbase.async.TestAsyncCheckAndMutate.getBuilder(TestAsyncCheckAndMutate.java:99)
	at com.google.cloud.bigtable.hbase.async.TestAsyncCheckAndMutate.checkAndDelete(TestAsyncCheckAndMutate.java:81)

Reply:
Please ignore NullPointerException, it is fixed.